src/components/Cart/CartItem.jsx

```diff
@@ -46,7 +46,9 @@ export const CartItem = ({ item }) => {
               {<FiPlus className='h-5 w-5 text-black-500 self-center' />}
             </Button>
           </div>
-          <p>{currencyFormatter((item.price * item.quantity).toFixed(2))}</p>
+          <p className='mt-3 lg:mt-0'>
+            {currencyFormatter((item.price * item.quantity).toFixed(2))}
```

Comment:
You don't need to use `toFixed` any more as it's already in the `currencyFormatter`